guides/common/modules/proc_configuring-smart-proxy-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,105 @@
+[id="configuring-{smart-proxy-context}-to-provision-{client-os-context}-on-Secure-Boot-enabled-hosts"]
+= Configuring {SmartProxy} to provision {client-os} on Secure Boot enabled hosts
+
+Secure Boot follows a chain of trust from the start of the host to the loading of Linux kernel modules.
+The first shim that is loaded determines which distribution can be booted or loaded by using a `kexec` system call until the next reboot.
+
+To provision {client-os} on Secure Boot enabled hosts with the *Grub2 UEFI SecureBoot* and *Grub2 UEFI HTTPS SecureBoot* PXE loaders, you have to provide signed shim and GRUB2 binaries provided by the vendor of your operating system.
+
+[IMPORTANT]
+====
+ifdef::satellite[]
+You have to perform the following configuration steps on each TFTP {SmartProxy} for a subnet to provision Secure Boot enabled hosts on that subnet.
+endif::[]
+ifndef::satellite[]
+You have to perform the following configuration steps on each TFTP proxy for a subnet to provision Secure Boot enabled hosts on that subnet.
```

Comment:
```suggestion
You have to perform the following configuration steps on each TFTP {SmartProxy} for a subnet to provision Secure Boot enabled hosts on that subnet.
```
Is there a reason not to call it SmartProxy in all builds?

Reply:
out of scope: in orcharhino, we have recently changes/fixed the branding in the WebUI. Changing this should not be part of this PR. Instead, I will eventually create a follow-up PR to remove some "ifdef::orcharhino -> use X proxy" in favor of "use X {SmartProxy}".

Please align with current docs.